src/screens/SendSheet.js

```diff
@@ -395,22 +399,35 @@ export default function SendSheet(props) {
       });
       analytics.track('Changed native currency input in Send flow');
     },
-    [maxInputBalance, selected.decimals, selected?.price?.value]
+    [maxEnabled, maxInputBalance, selected.decimals, selected?.price?.value]
   );
 
-  const sendMaxBalance = useCallback(async () => {
-    const newBalanceAmount = await updateMaxInputBalance(selected);
-    sendUpdateAssetAmount(newBalanceAmount);
-  }, [selected, sendUpdateAssetAmount, updateMaxInputBalance]);
+  useMemo(async () => {
```

Comment:
does this mean the useEffect on line 327 is no longer needed?

Reply:
also, interesting and somewhat unconventional way to use useMemo here instead of a useEffect, is that for any particular reason?